mediffusion/ddpm.py

```diff
@@ -161,6 +161,7 @@ def training_step(self, batch, batch_idx):
     def validation_step(self, batch, batch_idx):
         x_start = batch["img"]
         cls = batch["cls"] if self.class_conditioned else None
+        cls_log_indices = self.config.validation.cls_log_indices
```

Comment:
I suggest moving `cls_log_indices = self.config.validation.cls_log_indices` to the _log_image method as it is only going to be used there. We can then remove the `cls_log_indices` argument for that as it is going to be read from the config directly. What do you think?